packages/core/parcel-bundler/src/Server.js

```diff
@@ -107,6 +109,13 @@ function middleware(bundler) {
       res.writeHead(404);
       res.end();
     }
+
+    function logAccessIfVerbose() {
+      const protocol = req.connection.encrypted ? 'https' : 'http';
+      const fullUrl = `${protocol}://${req.headers.host}${req.url}`;
+
+      logger.verbose(fullUrl);
```

Comment:
Probably a good idea to add some more info here, besides from just a url, something like `Request: <url>`